pkg/resources/fluentd/configsecret.go

```diff
@@ -56,8 +56,13 @@ func (r *Reconciler) generateConfigSecret() (map[string][]byte, error) {
 	input := fluentdConfig{
 		IgnoreSameLogInterval:     r.Logging.Spec.FluentdSpec.IgnoreSameLogInterval,
 		IgnoreRepeatedLogInterval: r.Logging.Spec.FluentdSpec.IgnoreRepeatedLogInterval,
-		RootDir:                   r.Logging.Spec.FluentdSpec.RootDir,
 		EnableMsgpackTimeSupport:  r.Logging.Spec.FluentdSpec.EnableMsgpackTimeSupport,
+		Workers:                   r.Logging.Spec.FluentdSpec.Workers,
+	}
+
+	input.RootDir = r.Logging.Spec.FluentdSpec.RootDir
```

Comment:
This default value is computed at this level so I can use the already defined `bufferPath` variable. 
Let me know if you want me to move it into the `logging_types.go` file.

Reply:
It would make sense, thanks for pointing this out!
Could you please also move the loglevel default while you are here?

Reply:
Done with [`25f0a78` (#1417)](https://github.com/kube-logging/logging-operator/pull/1417/commits/25f0a7880062e17da06be64195d2dd86165f031c)